src/ArtifactBuildRestClient.ts

```diff
@@ -1,92 +1,33 @@
-import { IVssRestClientOptions } from 'azure-devops-extension-api';
-import { Build, BuildArtifact } from 'azure-devops-extension-api/Build'
-import { RestClientBase } from 'azure-devops-extension-api/Common/RestClientBase'
+import { BuildRestClient } from 'azure-devops-extension-api/Build'
+import { getAccessToken } from 'azure-devops-extension-sdk';
 
-export type ArtifactBuildRestClient = Pick<BuildRestClient2, 'getArtifacts' | 'getArtifactContentZip'>
+export type ArtifactBuildRestClient = Pick<BuildRestClient, 'getArtifacts'>
```

Comment:
>getArtifacts [](http://example.com/codeflow?start=61&length=12)

does getArtifacts being used? I see 
`export type ArtifactBuildRestClient = Pick<BuildRestClient, 'getArtifacts'>`

btw is `getBuild` being used?

Reply:
Yes, getArtifacts is used in getArtifactsFileEntries
getBuild is used in build.tsx

Reply:
but the changeset in this file looks like 
`public async getArtifacts(`
`public async getBuild(`
at the bottom are deleted